<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Midway Evaluation</title>
    <meta charset="utf-8" />
    <meta name="author" content="üòé Jonelle Dickow Villar" />
    <meta name="date" content="2021-11-25" />
    <script src="Midway_presentation_files/header-attrs/header-attrs.js"></script>
    <link href="Midway_presentation_files/tile-view/tile-view.css" rel="stylesheet" />
    <script src="Midway_presentation_files/tile-view/tile-view.js"></script>
    <link href="Midway_presentation_files/font-awesome/css/all.min.css" rel="stylesheet" />
    <link href="Midway_presentation_files/font-awesome/css/v4-shims.min.css" rel="stylesheet" />
    <link rel="stylesheet" href="xaringan-themer.css" type="text/css" />
    <link rel="stylesheet" href="yellow_dots.css" type="text/css" />
    <link rel="stylesheet" href="extra.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# Midway Evaluation
## Dept. of Clinical Science II <br/> University of Bergen
### üòé Jonelle Dickow Villar
### November 25, 2021

---






<div>
<style type="text/css">.xaringan-extra-logo {
width: 250px;
height: 250px;
z-index: 0;
background-image: url(UiBlogoMED_gray_v_Eng.png);
background-size: contain;
background-repeat: no-repeat;
position: absolute;
top:2em;right:2em;
}
</style>
<script>(function () {
  let tries = 0
  function addLogo () {
    if (typeof slideshow === 'undefined') {
      tries += 1
      if (tries < 10) {
        setTimeout(addLogo, 100)
      }
    } else {
      document.querySelectorAll('.remark-slide-content:not(.title-slide):not(.inverse):not(.hide_logo)')
        .forEach(function (slide) {
          const logo = document.createElement('div')
          logo.classList = 'xaringan-extra-logo'
          logo.href = null
          slide.appendChild(logo)
        })
    }
  }
  document.addEventListener('DOMContentLoaded', addLogo)
})()</script>
</div>


&lt;style type="text/css"&gt;

.small { 
  font-size: 0.75em
}  

.bitty { 
    font-size: 1.25em;
}

.enlarge { 
    font-size: 1.5em;
}

.fade {
  opacity: 0.66;
}

&lt;/style&gt;


---

class:hide_logo

###**An Epigenetic Study of Treatment Effects and Environmental Factors in Schizophrenia and Bipolar Disorder**&lt;br&gt;&lt;br&gt;

.large[.right[
**Principal Supervisor**: St√©phanie Le Hellard, UiB&lt;br&gt;&lt;br&gt;
**Co-Supervisors**: Anne-Kristin Stavrum, UiB,&lt;br&gt;&lt;br&gt;
                  Tetyana Zayats, Broad Institute, USA&lt;br&gt;&lt;br&gt;
                  Ingrid Melle, UiO&lt;br&gt;&lt;br&gt;&lt;br&gt;
                  July 1, 2019 - June 30, 2023
]]              

---

###Plan for this talk &lt;br&gt; &lt;br /&gt;
###- Objectives &lt;br /&gt;
###- Methods &lt;br /&gt;
###- Status of project &lt;br /&gt;


---

###Objectives of this PhD work &lt;br /&gt;

###**General aim**
.bitty[
 - To identify epigenetic marks associated with psychosis and treatment effects &lt;br /&gt;
]

###**Specific aims**
.bitty[
1. Association of SCZ vs. controls on cell-type proportions is impacted by time-of-blood draw &lt;br&gt; &lt;br /&gt;
2. Common and specific effects of antipsychotic drugs in psychosis &lt;br&gt; &lt;br /&gt;
3. Disorder specific effects of SCZ vs. BPD &lt;br&gt; &lt;br /&gt;
4. Evaluate whether any specific effects are associated with developmental timing 
]  

---

class: hide_logo

background-image: url(Smigielski.png)
background-position:cover
background-size: 750px

.footnote[Smigielski, _et al_, Molecular Psychiatry, 2020]



## **Epigenetics**



---


###Epigenetics in psychiatric disorders  &lt;br&gt; &lt;br&gt; 

.bitty[
- Development: prenatal environmental factors _in utero_ &lt;br&gt; &lt;br&gt;
- MZ twins discordant for SCZ/BPD (Dempster, _et al._, 2011) &lt;br&gt; &lt;br&gt;
- Dynamic; many environmental exposures - trauma, adversity, life-style &lt;br&gt; &lt;br&gt;
- Associated with brain volume, structure and function; &lt;br&gt; &lt;br&gt;
    - social and cognitive function (Starnawska &amp; Demontis, 2021) &lt;br&gt; &lt;br&gt;
- DNAm changes overtime as a function of age and chronicity of the disorder 
]


---

###Circadian rhythms and "chrono-epigenetics" &lt;br&gt; &lt;br&gt; 

.bitty[
- Methyl cycle co-evolved with circadian clocks (Fustin, _et al._, 2020) &lt;br&gt; &lt;br&gt;
- Circadian rhythms are disrupted in serious mental disorders (Walker II, _et al._, 2020) &lt;br&gt; &lt;br&gt;
- Chrono-epigenetics: &lt;br&gt; &lt;br&gt;
    - temporal dynamics of epigenetic processes (Oh, E. &amp; Petronis, 2021)
]

---


### **Epigenetic oscillations**&lt;br&gt; 

.pull-left[
![](Sim_officeHours.png)
]

.pull-right[

.bitty[
- figure b
 - blood collection time: 09-16
 - example time: 12:00 
 - black lines: oscillation profiles
 - red dots: collection time &lt;br&gt; &lt;br&gt;
- figure c
 - cpg density at 12:00 
 ]

]

.footnote[Oh, E. &amp; Petronis, Nature Reviews Genetics, 2021]


---

background-image: url(Daylight_EG.png)
background-position: 60% 75% 
background-size: 515px

class: left, top

###Project 1

###Does time-of-blood draw impact the association of SCZ vs controls on cell-type proportions?

.pull-right[.footnote[Eug√®ne Gabritschevsky, _Untitled_,1947]
]

---


###Methods: Pre-data analysis


.bitty[
1. DNA extraction from blood samples by FHI &lt;br&gt; &lt;br&gt;
1. State-of-the art microarray analysis: **Illumina EPIC 850K**  &lt;br&gt; &lt;br&gt;
1. QC: Three phases of TOP methylation data QCed separately and then merged &lt;br&gt; &lt;br&gt; 
    - Removal of probes with detection p-value &gt; 0.01, PCA, FunnNorm, SVD, Combat &lt;br&gt; &lt;br&gt;
1. Estimated cell-type proportions from DNAm data (Salas, _et al._, 2018) &lt;br&gt; &lt;br&gt;
1. Evaluated quality of cell-type estimation: &lt;br&gt; &lt;br&gt;
    - Deconvolution Specific Root Mean Squared Error (DSRMSE) (Vellame, _et al._, 2020) &lt;br&gt; &lt;br&gt;
1. Calculated smoking score: (Li, _et al._, 2018)
]

---

###Methods: Analysis pipeline 


.bitty[
1. Selected European SCZ cases &amp; controls with recorded time-of-blood draw (n=729) &lt;br&gt; &lt;br&gt;
1. Calculated variable ‚Äúhours from 07:00 baseline‚Äù &lt;br&gt; &lt;br&gt;
1. Samples matched for age and gender &lt;br&gt; &lt;br&gt;
1. Evaluated correlation between PCA and variables including batch effects &lt;br&gt; &lt;br&gt;
1. Performed linear regression on cell-type ~ AMP plate &lt;br&gt; &lt;br&gt;
1. Removed samples with DSRMSE &lt; 0.06 &lt;br&gt; &lt;br&gt;
1. Performed linear regression on residuals from Step 3 &lt;br&gt; &lt;br&gt;
1. Compared full day vs. half day &lt;br&gt; &lt;br&gt;
1. Sub-analyses: Neutrophil-to-lymphocyte ratio (NLR) and Medication-free SCZ cases

]

---

background-image: url(2021_08_05_Histogram_n729.png)
background-position: 50% 90% 
background-size: 425px

class: left, top
###Distribution of recorded blood draws (n=729)


- Model 1: Cell-type proportions ~  Case_Control + Age + Sex + Smoking_score + Methbatch + ùú∫
- Model 2: Cell-type proportions ~ Case_Control + Age + Sex + Smoking_score + Methbatch + Hours_from_baseline + ùú∫


---


background-image: url(2021_09_04_Table.CorrectPvalsTeal.png)
background-position: 25% 45% 
background-size: 400px

class: top left hide_logo
###Results

.pull-right[

###Full day (n=729)

.bitty[
- P-values reduced significance in Model 2
- Effect sizes diminished in Model 2
]

###Half day (n=524)

.bitty[
- P-values: no significant change 
- Effect sizes: no change
]
]

---
class: top left hide_logo
###Results Sub-analysis 

.bitty[
- NLR: &lt;br&gt; &lt;br&gt;
    - significant association in all analyses (_p_ &lt;.001); &lt;br&gt; &lt;br&gt;
    - modest increase in mean NLR in half-day medication-free samples; i.e. &lt;br&gt; &lt;br&gt;
      - unadjusted mean NLR (.93) vs. adjusted mean NLR  (1.14) &lt;br&gt; &lt;br&gt;
- Medication-free cases: significant for neutrophils and CD4T adjusted/unadjusted (_p_ &lt;.01)
]

### Conclusion Project 1:
.bitty[
Matching cases and controls for blood collection time reduces bias 
and the risk of overestimating the association in observational case-control studies 
due to circadian cycling of white blood cells. 
]

---

background-image: url(1947superJumbo.jpg)
background-position: 60% 75%
background-size: 525px

class: left, top
###Project 2
###Common and specific effects of antipsychotics associated with psychosis

.pull-left[
.footnote[Eug√®ne Gabritschevsky, _Untitled_,1947]
]

---

background-image: url(Aringhieri2.jpg)
background-position: 60% 75%
background-size: 775px

class: left, top

###Antipsychotics and corresponding receptors

.footnote[Aringhieri _et al_, Pharmacology &amp; Therapeutics, 2018]

---

### Pharmacoepigenetic findings in psychiatry 

.bitty[
- Early stages still: &lt;br&gt; &lt;br&gt;
  - Study design, technology, patient population, lack of replication &lt;br&gt; &lt;br&gt;
  - No uniform identification of hyper- or hypo-methylation (i.e. _COMT_, _5-HTT_, _HTR2A_) (Burghardt, _et al._, 2020, Zhou _et al_, 2021) &lt;br&gt; &lt;br&gt;
  - Lacking studies of AP monotherapy (Burghardt, _et al._, 2020) &lt;br&gt; &lt;br&gt;
  - DNAm associated with both the pathology and treatment (Jaffe, _et al._, 2016) &lt;br&gt; &lt;br&gt;
  - "Tissue issue" - relevance of peripheral blood (Bakulski, _et al._, 2016)
]

---

background-image: url(rho_table2.png)
background-position: 50% 70%
background-size: 800px


###Correlation blood-brain DNAm  &lt;br&gt; 

.bitty[
- Technological improvements:  450K vs. EPIC (850K)  
- Blood / live brain-tissue samples:      r = 0.86 
- CpGs identified within promoter: r = 0.92
]


.footnote[Braun _et al._, Translational Psychiatry, 2019]

---

###Methods &lt;br&gt; 

.bitty[
1. Select European SCZ, BPD and MDD cases being treated with antipsychotics &lt;br&gt; &lt;br&gt;
1. Select samples with blood drawn within 4 months of interview date &lt;br&gt; &lt;br&gt;
1. **Common**: Compare AP-use vs non AP-use   &lt;br&gt; &lt;br&gt;
1. **Specific**: Contrast mono-therapy against non-target polytherapy.  &lt;br&gt; &lt;br&gt;
1. Model: DNAm ~ Psychosis_Cases + Age + Sex + Smoking + eCells + Technical + ùú∫&lt;br&gt; &lt;br&gt;
1. Identify CpGs (_limma_), associated genes (_Illumina annotation), and DMRs (_Demarcate_) &lt;br&gt; &lt;br&gt;
1. GSEA and pathway analysis (GREAT) &lt;br&gt; &lt;br&gt;

]


---


background-image: url(Sans-titre-1949-Gouache-sur-papier.jpg)
background-position: 50% 70%
background-size: 525px


class: left, top 

###Project 3
###Identify differential DNA methylation in SCZ versus Bipolar

.pull-left[
.footnote[Eug√®ne Gabritschevsky, _Untitled_,1949]
]

---
###Characteristics 

###Similarities

.bitty[
- Early age of onset ~ 20-30 years old &lt;br&gt; &lt;br&gt;
- Heritability ~ 80% &lt;br&gt; &lt;br&gt;
- Periods of stability and relapse &lt;br&gt; &lt;br&gt;
- Dysregulated circadian rhythmicity: both a trait and state marker (Walker II, _et al._, 2020) &lt;br&gt; &lt;br&gt;
- Medication primary form of treatment 
]

---

### Differences &lt;br&gt; &lt;br&gt;

.bitty[
- Polygenic risk score (PRS) differed significantly (Ruderfer, _et al., 2014) &lt;br&gt; &lt;br&gt;
- Two genome-wide significant SNPs (BPD&amp;SCZ Working Group of the PGC., 2018) &lt;br&gt; &lt;br&gt;
- Clinical differences behavioral, cognition, intelligence (Jabben,_et al._, 2010, Parellada, _et al._, 2017) &lt;br&gt; &lt;br&gt; 
]

---

class: top left
### **Clinical Course of SCZ and BPD**

.pull-left[
![](ClinicalCourse_adapt.png)
]

.pull-right[
![](course_BPD.jpeg)
]


.pull.left[.footnote[McCutcheon, _et al.,_ JAMA Psychiatry, 2020]]
.pull-right[.footnote[Elanjithara, _et al.,_ 2011]]


---

###Methods &lt;br&gt; &lt;br&gt;

.enlarge[
1. Select SCZ and BPD cases from Project 2 &lt;br&gt; &lt;br&gt;
1. Extract CpGs associated with antipsychotics from Project 2 &lt;br&gt; &lt;br&gt;
1. Model: DNAm ~ SCZ_BDP + Age + Sex + Smoking + Cells + Technical +ùú∫&lt;br&gt; &lt;br&gt;
1. GSEA and pathway analyses (GREAT) &lt;br&gt; &lt;br&gt;
1. Compare annotated genes to GWAS findings
]

---

### References

.bitty[
Aringhieri, _et al_, Molecular targets of atypical antipsychotics: From mechanism of action to clinical differences, _Pharmacology &amp; Therapeutics_, 2018.

Bakulski, _et al._, Epigenetic Research in Neuropsychiatric Disorders: the ‚ÄúTissue Issue‚Äù, _Curr Behav Neurosci Rep., 2016.

Braun, _et al._, Genome-wide DNA methylation comparison between live human brain and peripheral tissues within individuals, _Translational Psychiatry_, 2019.

Dempster, _et al._, Disease-associated epigenetic changes in monozygotic twins discordant for schizophrenia and bipolar disorder, _Human Molecular Genetics_, 2011.

Elanjithara, _et al._, Treatment of early stages of bipolar disorder, _Advances in Psychiatric Treatment_, 2011.

]

---

### References cont.

.bitty[

Fustin, _et al._, Methylation deficiency disrupts biological rhythms from bacteria to humans, _Communications Biology_, 2020.

Jabben, _et al._, Neurocognitive functioning as intermediary phenotype and predictor of psychosocial functioning across the psychosis continuum: studies in schizophrenia and bipolar disorder, _J Clin Psychiatry_, 2010.

Jaffe &amp; Irizarry, Accounting for cellular heterogeneity is critical in epigenome-wide association studies, _Genome Biology, 2014.

Lappalainen &amp; Greally, Associating cellular epigenetic models with human phenotypes, _Nature Rev Genetics_, 2017.

Lee, _et al._, Disrupted-in-schizophrenia 1 enhances the quality of circadian rhythm by stabilizing BMAL1, _Translational Psychiatry_, 2021.


]

---
### References cont.

.bitty[

Li, _et al._, Causal effect of smoking on DNA methylation in peripheral blood: a twin and family study, _Clinical Epigenetics_, 2018.

McCutcheon, _et al._, Schizophrenia - An Overview, _JAMA Psychiatry_, 2019.

Oh, E. &amp; Petronis, Origins of human disease: the chrono-epigenetic perspective _Nature Rev Genetics_, 2021.

Parellada, _et al._, Developmental Differences Between Schizophrenia and Bipolar Disorder, _Schizophr. Bull._, 2017.

Rakyan, _et al._, Epigenome-wide association studies for common human diseases, _Nature Rev Genetics_, 2011.


]

---

### References cont.

.bitty[

Salas, _et al._, An optimized library for reference-based deconvolution of whole-blood biospecimens assayed using the Illumina HumanMethylationEPIC BeadArray, _Genome Biology_, 2018. 

Smigielski, _et al_, Epigenetic mechanisms in schizophrenia and other psychotic disorders: a systematic review of empirical human Ô¨Åndings, _Molecular Psychiatry_, 2020.

Vellame, _et al._, https://github.com/ds420/CETYGO.

Walker II, _et al._, Circadian rhythm disruption and mental health, _Translational Psychiatry_, 2020.

Wei, _et al._, Ten Years of EWAS, _Advanced Science_, 2021.

]

---

class: center, top

background-image: url(Martens_lab_2021.jpg)
background-position: 12% 70%, 20% 10%
background-size: 600px

# Acknowledgements

.pull-right[
**Supervisor Prof. St√©phanie Le Hellard**&lt;br&gt;&lt;br&gt;
**Co-Supervisor Dr. Anne-Kristin Stavrum**&lt;br&gt;&lt;br&gt;
**Colleagues in The Martens Group**&lt;br&gt;&lt;br&gt;
**Dept. of Clinical Science II** &lt;br&gt;&lt;br&gt;&lt;br&gt;
**NORMENT** &lt;br&gt;&lt;br&gt;&lt;br&gt;

[<i class="fab fa-twitter" role="presentation" aria-label="twitter icon"></i> @jdvillar](https://twitter.com/jdvillar)
]]


---


class: left top inverse 
#Conclusion of the public event &lt;br&gt; &lt;br&gt;

##Happy to take questions



---


background-image: url(ECT3.png)
background-position: 50% 65%
background-size: 600px

class: left top 
##Formal training components completed

---

background-image: url(posters3.png)
background-position: contain
background-size: 800px


class: left top 
##FORMIDL901 to be completed

---

background-image: url(Plan3.png)
background-position: 50% 65%
background-size: 675px

class: left top 
##Plan for project completion

---


class: left top 

###Technical Challenges &lt;br&gt;

.bitty[
- Currently no standardized pipeline for QC and EWAS (including DMRs and pathway analyses). &lt;br&gt;&lt;br&gt;
- EPIC probes biased for cancer genes. &lt;br&gt; &lt;br&gt;
- EWAS literature (2011- 2020): &lt;br&gt;&lt;br&gt;
    - 47% pertains to Illumina 450K; &lt;br&gt;&lt;br&gt;
    - &lt; 7% pertains to EPIC (Wei _et al.,_ 2021) &lt;br&gt;&lt;br&gt;
- Small sample sizes may lack power. &lt;br&gt; &lt;br&gt;
- Large meta-analyses may have different study designs or methods. 
]
 

---

class: left top 

###Scientific Challenges &lt;br&gt; &lt;br&gt;

.bitty[
- We can¬¥t control for all environmental factors. &lt;br&gt; &lt;br&gt;
- The data available to us is derived from one blood sample. &lt;br&gt; &lt;br&gt;
- Small sample sizes assayed on the chip provide analysis challenges (i.e. singularity) &lt;br&gt; &lt;br&gt;

]











    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"slideNumberFormat": "%current%",
"highlightStyle": "github",
"highlightLines": true,
"ratio": "16:9",
"countIncrementalSlides": true
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
